org.eclipse.scanning.test/src/org/eclipse/scanning/test/epics/EpicsV4ConnectorTest.java

```diff
@@ -163,9 +172,9 @@ public void connectToInvalidDevice() throws Exception {
 	public void connectToInvalidDeviceTimeout() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)